tests/utils/api-gateway/index.js

```diff
@@ -0,0 +1,63 @@
+const AWS = require('aws-sdk');
```

Comment:
`'use strict'` here and in other introduced modules (technically we should have lint rule turned on for that)